src/test/java/apoc/export/cypher/ExportCypherResults.java

```diff
@@ -0,0 +1,168 @@
+package apoc.export.cypher;
```

Comment:
why do we pull it out to a separate file? is it reused in some other place?
Otherwise makes it harder to keep test + expected results together?

Reply:
I get that the file gets quite long, we can push these also to the end of the file if that makes it more readable.